.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,60 @@
+name: CI
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        php: [7.1, 7.2, 7.3, 7.4]
+        env: ['EXECUTOR= DEPENDENCIES=--prefer-lowest', 'EXECUTOR=coroutine DEPENDENCIES=--prefer-lowest', 'EXECUTOR=', 'EXECUTOR=coroutine']
+        include:
+          - os: ubuntu-18.04
+            php: 7.1
+            env: EXECUTOR=coroutine
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install PHP
+      uses: shivammathur/setup-php@1.5.2
+      with:
+        php-version: ${{ matrix.php  }}
+        extension-csv: dom, mbstring
+        coverage: xdebug
+    - name: Install dependencies
+      run: |
+        export $ENV
+        echo ${DEPENDENCIES}
+        composer update ${DEPENDENCIES}
```

Comment:
Why update? 🤔

Reply:
Is it not actual

Reply:
install should be good enough as composer.lock is not versioned in libraries

Reply:
https://github.com/webonyx/graphql-php/blob/master/.travis.yml#L27 current travis build uses `composer update`

Reply:
Oh, my bad! Then lets keep it and don't change it 👍